harness/determined/common/schemas/expconf/_gen.py

```diff
@@ -2085,6 +2085,14 @@
             ],
             "default": null
         },
+        "slots_per_node": {
+            "type": [
+                "integer",
+                "null"
+            ],
+            "minimum": 1,
+            "default": 1
```

Comment:
I think maybe the default should be null and we should have custom validation to ensure it is set for pbs. @rb-determined-ai thoughts?

Reply:
I did wonder if the default might be better as null -- such that we can tell if it were present or not, but I wasn't convinced that it mattered to the result.   Might be a bit clearer though.

It doesn't need to be set -- it just may be to use more than 1 GPUs when there is no gres configured (Slurm or PBS).

[AT-244122801-200922-1715-48.pdf](https://github.com/determined-ai/determined/files/9626059/AT-244122801-200922-1715-48.pdf)

Reply:
I was thinking we would pay attention to it only when > default